nginx/conf.d/default.conf.template

```diff
@@ -11,16 +11,4 @@ server {
     location = /50x.html {
         root   /usr/share/nginx/html;
     }
-}
-
-server {
-    listen $PORT;
-    server_name api.localhost;
-
-    location / {
-        proxy_pass  http://localhost:8080/v1/;
-        proxy_set_header Host $host;
-    }
-
-
```

Comment:
Don't add or remove anything from this file leave it as it was. A pr should be focused and changing nginx config in a pr which focuses on models and database related stuff violates that.